test/e2e/storage/drivers/csi.go

```diff
@@ -61,6 +61,11 @@ func initHostPathCSIDriver(name string, config testsuites.TestConfig, manifests
 			Name:        name,
 			FeatureTag:  "",
 			MaxFileSize: testpatterns.FileSizeMedium,
+			SupportedSizeRange: framework.SizeRange{
+				Min:   float64(1), // TODO: choose appropriate values
```

Comment:
As for the values for ```Min``` and ```Max```,  we can safely choose below defaults:
[Each testsuite]
- Min: "1Mi"         ... Requested minimum requirement from driver side.
- Max: "10Gi"       ... More than or equal to known driver's maximum (5Gi).

[Each Driver]
- Min: "1Gi"          ... Original default size of the tests (Before increasing it to 5Gi)
- Max: "10Gi"       ... The same to testsuites' default maximum.

In addition, we need to add below exception:
[CSI GCE drivers]
- Min: "5Gi"          

https://github.com/kubernetes/kubernetes/blob/master/test/e2e/storage/drivers/csi.go#L173
https://github.com/kubernetes/kubernetes/blob/master/test/e2e/storage/drivers/csi.go#L267